client/src/App.js

```diff
@@ -3,8 +3,8 @@ import { AppContext } from "./AppContext";
 import { useState } from "react";
 import "./App.css";
 
-import Home from "./pages/Home";
-import Result from "./pages/Result";
+import {Home} from "./pages/Home";
+import {Result} from "./pages/Result";
```

Comment:
I would have thought you formating would have put spaces here eg
`{ Home }` rather than `{Home}`